modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -693,7 +693,7 @@ export default {
           return true;
         }
       }
-      return false
+      return false;
```

Comment:
Go with the filter pattern — do `if (!url)...` up front so that most of the function does not have to be indented.

Reply:
I updated the entire function since I was at it, including your suggestion 👍